src/Stack/Build/Precompiled.hs

```diff
@@ -0,0 +1,198 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Cache management during a build.
+-- Handle all the pre-installed, precompiled & prebuilt stuff.
+module Stack.Build.Precompiled
+(
+    copyPreCompiled
+    , getPrecompiled
+    , loadInstalledPkg
+    , getExecutableBuildStatuses
+) where
+
+import           Stack.Prelude
+import           Stack.GhcPkg
+import           Stack.Types.Build
+import           Stack.Types.Package
+import           Stack.Types.Config
+import           Stack.Types.Execute
+import           Stack.Types.GhcPkgId
+import           Stack.Build.Cache
+import           Stack.PackageDump (conduitDumpPackage, ghcPkgDescribe)
```

Comment:
Stack's code base has different ways of formatting imports in different modules but I think it's better at least not to mix different ways in 1 module